aries-site/src/pages/components/dateinput.mdx

```diff
@@ -0,0 +1,75 @@
+import { Example } from '../../layouts';
+import {
+  DateInputRangeExample,
+  DateInputExample,
+  DateInputSimpleExample,
+  DateInputValidationExample,
+} from '../../examples/components/dateinput';
+
+
+<Example
+  docs="https://v2.grommet.io/dateinput?theme=hpe"
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/dateinput/DateInputSimpleExample.js"
+  figma="https://www.figma.com/file/Mf9if6zyGETTWW5b9PsYNQ/HPE-Date-Input-Component?node-id=1%3A567"
+>
+  <DateInputSimpleExample />
+</Example>
+
+## Guidance
+
+- DateInputs can be used to display past, present, or future dates or times.
+- Use dateinput when you are asking the user for a time or date, or for scheduling tasks.
+
+### When using Date Input, you should:
+
+* Use a label when appropriate
+* Correctly format to reduce mistakes such as having the caller specify
+  the appropriate date format for the locale the user is in. 
+* For example, in the U.S. dates are formatted as ‘mm/dd/yyyy’ whereas
+ in other countries, such as Europe it might be ‘dd/mm/yyyy’.
+
+### Usage
```

Comment:
Feel like this "Usage" subheading isn't very necessary since this sentence was covered under "Guidance"

Reply:
We could also remove the bullet below it